docs/docs/css/extra.css

```diff
@@ -10,6 +28,21 @@
     color: #222;
 }
 
+.md-typeset code {
+    font-size: .8em;
+}
+
+.md-header__title {
+    font-family: "Space Mono";
+    font-weight: 400;
+    font-style: normal;
+    font-size: 1.5rem;
+}
+
+.md-typeset > p, .md-typeset > ul, .md-typeset > ol, .md-typeset > blockquote, .md-typeset > div.admonition {
+    max-width: 75%;
+}
```

Comment:
This leads to a bit of strange behavior on large screens and small screens:
Large screens: different widths of certain elements
![image](https://github.com/drivendata/cookiecutter-data-science/assets/23062974/c90c9d5e-a7e0-4cfe-8498-a22c830890a5)

Small screens: text not full width
![image](https://github.com/drivendata/cookiecutter-data-science/assets/23062974/5a48bc6c-0a78-4ea7-b7ef-f51353b28e81)

IMO, if we don't want things to be so wide, we should instead set the max-width of the content (and maybe header/footer) in pixels.

Reply:
Resolved by setting max width in em